pre-commit

```diff
@@ -272,7 +272,7 @@ def load_lilac_yaml_schema():
         YAML_SCHEMA = yaml.load(schema, Loader=yaml.SafeLoader)
 
 
-def iter_packages():
+def iter_packages(all_pkgs: bool = False):
```

Comment:
Return type not specified; mypy will skip.

Reply:
Thanks I often forget return types :sweat_smile:  Fixed now.